### PR TITLE
Qualcomm updates

### PR DESCRIPTION
**Pull request by JamiKettunen**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by JamiKettunen** [comment]
Sorry for yet another force push, had to sort out the messed up authorship; I prepared a patch for LKML earlier so I had the wrong gitconfig setup.

**Comment by Duncaen** [comment]
Is there a reason to split out the qcom firmware?
Is it possible that users depend qcom firmware and splitting it out without anything depending on that package results in users not having required firmware available?

**Comment by JamiKettunen** [comment]
> Is there a reason to split out the qcom firmware?

I thought it would be nice to get a ~20 MB subpackage for users that needed these instead of a ~104 MB one containing everything else except the other few split packages.

Sadly my platform (MSM8998) doesn't have many pieces in `linux-firmware` in general, and due to the smartphone manufacturer I can only use a few files (non-zap Adreno FW bits) from the subpackage and the rest need to be packaged separately with their OEM signature on top of the Qualcomm one.
> Is it possible that users depend qcom firmware and splitting it out without anything depending on that package results in users not having required firmware available?

That could be a case, though as the officially supported platforms ([as per current live ARM rootfs'es](https://alpha.de.repo.voidlinux.org/live/current/)) don't include a single Qualcomm SoC this likely won't be a problem.

File: srcpkgs/linux-firmware/template [hunk 1]
```diff
@@ -94,3 +95,11 @@ linux-firmware-network_package() {
 		vmove usr/lib/firmware/qca
 	}
 }
+
+linux-firmware-qcom_package() {
```
Comment (ericonr) [inline srcpkgs/linux-firmware/template:99]: Nit: why `qcom` and not `qualcomm`?

**Review by ericonr — COMMENTED** [review]
If you want to make merging this simpler, I think we can split the firmware change into a separate PR. What do you think?

Please rebase the PR too :)

File: srcpkgs/linux-firmware/template (hunk 1 above)
Comment (JamiKettunen) [reply srcpkgs/linux-firmware/template:99]: Matches the `linux-firmware` directory name and is already done the same way on e.g. [Alpine Linux](https://pkgs.alpinelinux.org/packages?name=linux-firmware-qcom).

Comment (JamiKettunen) [reply srcpkgs/linux-firmware/template:99]: I can certainly split the FW subpackage to another PR and rebase this, just a few minutes.

**Pull request merged**